app/services/food_service.rb

```diff
@@ -0,0 +1,34 @@
+class FoodService
+  def self.search_foods(search)
+    conn = Faraday.new(url: 'https://api.nal.usda.gov') do |faraday|
+      # faraday.headers['api_key'] = ENV['movie_api_key']
+    end
+    query = search
+    # query[' '] = '%20'
+    query.gsub(' ', "%20")
+
+    response = conn.get("/fdc/v1/foods/search?api_key=#{ENV['api_key']}&query=#{search}")
+
+    if response.status == 200
+      results = JSON.parse(response.body, symbolize_names: true)
+      # session[:total_results] = results[:totalHits]
```

Comment:
Instead of having to leverage the session here you have a few options:
1) pass back the whole response hash and extract the total hits/foods area in teh facade
2) pass back both total hits and foods array back to the facade